### ISSUE TITLE
Zepto issue

### ISSUE THREAD

**Issue by YannickForest** [Pagawa/PgwModal#18]
Need to remove the container created by getScrollbarWidth() when using Zepto. Add:

```javascript 
container.remove();
```
right after line 120.

**Comment by mauron85** [comment]
Yes, please do. Because when using zepto, there will be container leftovers in the DOM.
